Documentation/ExtensionArchitecture/FrontendPlugin/AbstractPlugin.rst

```diff
@@ -0,0 +1,126 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Extbase; AbstractPlugin
+
+.. _abstractplugin:
+
+==============
+AbstractPlugin
+==============
```

Comment:
```suggestion
==============================
AbstractPlugin (aka `pi_base`)
==============================
```

Reply:
This messes up the Menu and sounds very colloquial. I have mentioned the name below.

Reply:
AFAIS the literal term `pi_base` is not used so I'm not sure if it will be found by our global search and search engines. If you are sure, this will work, then we can skip it. Otherwise it should be mentioned explicitly. People with legacy codebase should be able to find this even without checking the TYPO3 changelog/history.

Reply:
See the added section about the old name

Reply:
Love it, thanks!